apollo/src/schema.js

```diff
@@ -112,6 +129,12 @@ const typeDefs = gql`
     rating: Int!
   }
 
+  type Label {
```

Comment:
added Label type but didn't add CCRepo type in apollo

Reply:
CCRepo not our bag

Reply:
okay just make sure apollo has types that it can also understand otherwise it will cause issues

Reply:
> CCRepo not our bag

Curious to why CCRepo is there then. You'll need to remove it if it isn't associated with your feature.

Reply:
not sure. I did not add it but i will take it out with, hopefully, the final push